.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: Integration Tests
+
+'on':
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  check-author:
+    runs-on: ubuntu-latest
+    outputs:
+      is-trusted: ${{ steps.check.outputs.trusted }}
+    steps:
+      - name: Check if author is trusted
```

Comment:
I configured an approval flow for running actions in the repo settings. We don't need this check anymore